test/index.test.js

```diff
@@ -327,12 +327,12 @@ describe('JWT token decoding', function () {
   it('should complain if the HTTP Authorization header is in the wrong format', async function () {
     const response = await server.inject({ method: 'GET', url: '/decode', headers: { Authorization: 'FOO' } })
 
-    expect(response.statusCode).toEqual(400)
+    expect(response.statusCode).toEqual(401)
```

Comment:
This feels like oauth2-mock-server matured its error messages but I need to dig into to make sure.  Hence the draft.

Reply:
These two test changes are the result of upgrading @fastify/jwt.

REL: https://github.com/fastify/fastify-jwt/pull/325